ycms/cms/templates/_base.html

```diff
@@ -85,7 +53,73 @@
             {% endfor %}
         </div>
     {% endif %}
-    <div class="py-6 mt-24 mx-12">
+    <div id="sidebar" class="absolute h-full w-64 bg-white border-r">
+        <a href="{% url 'cms:protected:index' %}" class="block border-b">
+            <img src="{% static 'images/logo-ycms.png' %}"
+                 width="200px"
+                 height="auto"
+                 alt="{% translate " Logo" %}"
+                 class="mx-auto">
+        </a>
+        <ul>
+            <span class="block mt-5 px-5 text-gray-500 tracking-wide">{% translate "General" %}</span>
+            <li>
+                {% url 'cms:protected:zbm' as url %}
+                <a href="{{ url }}"
+                   class="{% if request.path == url %}selected{% endif %}">
+                    <i icon-name="activity"></i> {% translate "ZBM View" %}
+                </a>
+            </li>
+            <li>
+                {% url 'cms:protected:index' as url %}
+                <a href="{{ url }}"
+                   class="{% if request.path == url %}selected{% endif %}">
+                    <i icon-name="folder-search"></i> {% translate "Manage Patients" %}
+                </a>
+            </li>
+            <span class="block mt-5 px-5 text-gray-500 tracking-wide">{% translate "Ward" %}</span>
+            <li>
+                {% url 'cms:protected:ward_detail' pk=1 as url %}
```

Comment:
```suggestion
                {% url 'cms:protected:ward_detail' pk=user.assigned_ward.id|default:1 as url %}
```
I know we only have one ward currently, but then this will work ootb later :D